prow/github/fakegithub/fakegithub.go

```diff
@@ -22,7 +22,9 @@ import (
 	"fmt"
 	"regexp"
 
```

Comment:
It doesn't look like you are using any of the existing features of fakegithub and are only using the new APIs you are adding. Since no other components will need these APIs please write a separate, targeted fake instead of adding this to the shared fakegithub package.

Reply:
I have moved it to a separate package, which is now `prow/cmd/hmac/fakeghhook`.